app/styles/application.scss

```
@@ -1,79 +1,16 @@
----
-eleventyExcludeFromCollections: true
----
+@use "pkg:nhsuk-frontend/dist/nhsuk" as *;
 
-@font-face {
-  font-display: swap;
-  font-family: "Frutiger W01";
-  font-style: normal;
-  font-weight: 400;
-  src: url("https://assets.nhs.uk/fonts/FrutigerLTW01-55Roman.eot?#iefix");
-  src: url("https://assets.nhs.uk/fonts/FrutigerLTW01-55Roman.eot?#iefix") format("eot"), url("https://assets.nhs.uk/fonts/FrutigerLTW01-55Roman.woff2") format("woff2"), url("https://assets.nhs.uk/fonts/FrutigerLTW01-55Roman.woff") format("woff"), url("https://assets.nhs.uk/fonts/FrutigerLTW01-55Roman.ttf") format("truetype"), url("https://assets.nhs.uk/fonts/FrutigerLTW01-55Roman.svg#7def0e34-f28d-434f-b2ec-472bde847115") format("svg");
-}
-
-@font-face {
-  font-display: swap;
-  font-family: "Frutiger W01";
-  font-style: normal;
-  font-weight: 600;
-  src: url("https://assets.nhs.uk/fonts/FrutigerLTW01-65Bold.eot?#iefix");
-  src: url("https://assets.nhs.uk/fonts/FrutigerLTW01-65Bold.eot?#iefix") format("eot"), url("https://assets.nhs.uk/fonts/FrutigerLTW01-65Bold.woff2") format("woff2"), url("https://assets.nhs.uk/fonts/FrutigerLTW01-65Bold.woff") format("woff"), url("https://assets.nhs.uk/fonts/FrutigerLTW01-65Bold.ttf") format("truetype"), url("https://assets.nhs.uk/fonts/FrutigerLTW01-65Bold.svg#eae74276-dd78-47e4-9b27-dac81c3411ca") format("svg");
-}
+@use "../_components";
```

Should this be `@forward`?

```suggestion
@forward "../_components";
```

<br>

Same with the nested loads inside?

```patch
- @use "email";
- @use "screenshots";
+ @forward "email";
+ @ forward "screenshots";
```